libraries/matrix/api/src/main/kotlin/io/element/android/libraries/matrix/api/permalink/MatrixToConverter.kt

```diff
@@ -17,40 +17,18 @@
 package io.element.android.libraries.matrix.api.permalink
 
 import android.net.Uri
-import io.element.android.appconfig.MatrixConfiguration
 
 /**
  * Mapping of an input URI to a matrix.to compliant URI.
  */
-object MatrixToConverter {
+interface MatrixToConverter {
     /**
      * Try to convert a URL from an element web instance or from a client permalink to a matrix.to url.
-     * To be successfully converted, URL path should contain one of the [SUPPORTED_PATHS].
+     * To be successfully converted, URL path should contain one of the [DefaultMatrixToConverter.SUPPORTED_PATHS].
```

Comment:
Shouldn't we move the constant here, or maybe not mention the concrete implementation?

Reply:
I have removed the comment.